src/execution/const_interpreter_loop.rs

```diff
@@ -95,6 +101,23 @@ pub(crate) fn run_const(
                 trace!("Constant instruction: i64.mul [{v1} {v2}] -> [{res}]");
                 stack.push_value(res.into());
             }
+            REF_NULL => {
+                let reftype = RefType::read_unvalidated(&mut wasm);
+
+                stack.push_value(Value::Ref(reftype.to_null_ref()));
+                trace!("Instruction: ref.null '{}' -> [{}]", reftype, reftype);
+            }
+            REF_FUNC => {
+                // not unwrap validated because, guess what? not validated

```

Comment:
Why is this not validated?

Reply:
You are right, it should be validated because we do validation first using `read_constant_instructions`